app/controllers/admin/merchants_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Admin::MerchantsController < Admin::BaseController
+  def index 
+    @merchants = Merchant.all
+  end
+
+  def show 
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update 
+    merchant = Merchant.find(params[:id])
```

Comment:
probably want to break this method out into helper methods during refactor phase